packages/browser/core/src/core/auth/StitchUser.ts

```diff
@@ -14,9 +14,45 @@
  * limitations under the License.
  */
 
-import { CoreStitchUser, StitchCredential } from "mongodb-stitch-core-sdk";
+import { Codec, CoreStitchUser, StitchCredential } from "mongodb-stitch-core-sdk";
```

Comment:
I don't understand why these changes were made.

Reply:
We're importing `Codec` now for the StitchUserCodec

Reply:
See rest of review as to why these changes are not necessary

Reply:
👍 removed